src/cmdline/mod.rs

```diff
@@ -167,8 +167,11 @@ pub enum PackCmd {
     /// Pack a minimal ISO into a CoreOS live ISO image
     MinimalIso(PackMinimalIsoConfig),
     /// Generate man pages for coreos-installer
-    #[cfg(feature = "mangen")]
+    #[cfg(feature = "docgen")]
     Man(PackManConfig),
+    /// Generate example config file for install subcommand
+    #[cfg(feature = "docgen")]
+    ExampleConfig(PackExampleConfigConfig),
```

Comment:
Looking at the CI failure, was this enum variant supposed to be `ConfigFormat` instead?

Reply:
Whoops, that was an incomplete refactor.  Should be fixed now.